ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DefendantMapper.java

```diff
@@ -0,0 +1,122 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDDefendant;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPartyType;
+import uk.gov.hmcts.cmc.ccd.exception.MappingException;
+import uk.gov.hmcts.cmc.domain.models.otherparty.CompanyDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.IndividualDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.OrganisationDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.SoleTraderDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+
+import java.time.LocalDate;
+
+import static java.time.format.DateTimeFormatter.ISO_DATE;
+
+@Component
+public class DefendantMapper implements Mapper<CCDDefendant, TheirDetails> {
```

Comment:
As defendant is not just the party details object, it has lot more things from case, please use BuilderMapper interface here.